test_scripts/Capabilities/UpdateVideoStreamingCapabilities/OnAppCapabilityUpdated/002_OnAppCapabilityUpdated_from_mobile_to_HMI_capability_parameter_non_mandatory_parameter_is_missed.lua

```diff
@@ -0,0 +1,41 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0296-Update-video-streaming-capabilities-during-ignition-cycle.md
+--
+-- Description:
+-- Processing OnAppCapabilityUpdated notification with only mandatory parameters from mobile to HMI
+--
+-- Precondition:
+-- 1. SDL and HMI are started
+-- 2. App with `PROJECTION` or `NAVIGATION` appHMIType and 5 protocol version is registered
+-- 3. OnAppCapabilityUpdated notification is allowed by policy for App
+--
+-- Sequence:
```

Comment:
The `appCapabilityType` param description states
```
The corresponding param to this enum should be included and the only other param included
```
Should the notification still be forwarded to the HMI, if `appCapabilityType` is `VIDEO_STREAMING` and the `videoStreamingCapability` is undefined?

Reply:
Yes, the notification should be forwarded to HMI in this case because SDL has no specific logic for the `OnAppCapabilityUpdated` notification. 
Validation is performed only according to API and according to the API the notification, in this case, is valid and should be forwarded to HMI.